rocAL/rocAL_pybind/examples/tf_petsTrainingExample/README.md

```diff
@@ -10,5 +16,5 @@ Use the instructions in the [docker section](../../../../../docker) to build the
 
 To run this example for the first run or subsequent runs, just execute:
 ```
-python3.9 train_withROCAL_withTFRecordReader.py
+python3 train_withROCAL_withTFRecordReader.py
```

Comment:
Please check if this changes are needed in this PR

Reply:
A change from ToT not reflected in Shobana's fork